src/main/java/com/blackducksoftware/integration/fortify/batch/step/BlackDuckFortifyPushThread.java

```diff
@@ -80,8 +78,11 @@
 
     private final static Logger logger = Logger.getLogger(BlackDuckFortifyPushThread.class);
 
+    private SpringConfiguration springConfiguration;
```

Comment:
Actually, is this class - BlackDuckForfifyPushThread a spring bean as well? I wouldn't expect a class to make an instance of the spring config. I'd expect the spring config to send its beans to the constructor of other beans. If this class isn't a bean, some bean must make it, in which case, i'd send the required instances to that bean and into this constructor.